pkgs/by-name/cl/clipse/package.nix

```
@@ -0,0 +1,28 @@
+{ 
+  lib, 
+  buildGoModule, 
+  fetchFromGitHub 
+}:
+
+buildGoModule {
```

Allow re-use of variables…
```suggestion
buildGoModule rec {
```